testing.ini.example

```diff
@@ -45,6 +45,7 @@ db_schema = public
 # For virtuoso:
 #db_schema = assembl_test
 db_database = assembl_test
+# db_host = localhost
```

Comment:
I think this is one change that came from your cherry-picking of changesets. Not intended for this PR, but will be merged once the rest of SAML branch is merged?